managers/asset-operations-manager.js

```diff
@@ -346,53 +346,34 @@ class AssetOperationsManager {
         const { tokenId } = resolveUAL(UAL);
 
         let publicAssertionId;
-        let isLatestUnfinalized = false;
-        if (state === ASSET_STATES.LATEST) {
+        if(state === ASSET_STATES.LATEST || /^0x[a-fA-F0-9]{64}$/.test(state)) {
```

Comment:
All hashes will pass this condition, even if it's a finalized state (we don't need to get pending state in this case)